string/readme.md

```diff
@@ -13,10 +13,19 @@ A string is a class that contains a char array, but automatically manages it for
 
 ## Questions :
 
+
+* Print the decimal equivalent of a binary string ---->[C++](/Code/C++/binary_string_to_decimal.cpp)
+
 * Convert string to lowercase and uppercase ---->[C++](/Code/C++/string_to_lowercase_and_uppercase.cpp)
 * Manacher's algorithm (finding the longest palindrome in a string) ----> [Java](/Code/Java/longPalindrome.java)
 * Maximum occuring character in a string ---->[C++](/Code/C++/max_count.cpp)
 * Number of deletions to make a string pallindrome ---->[Python](/string/Number_of_deletions_to_make_pallindrome.py)
+
 * Revere a string using Stack ---->[C++](/Code/C++/reverse_a_string_using_stack.cpp)
 * Reverse Individual Words in String ----> [Java](/Code/Java/revindivstring.java)
```

Comment:
sort this list in lexicographic order

Reply:
@rish-singhal  Sir I have done as asked...just one question..there are two duplicate codes in this readme ..will I remove the duplicates?